orc8r-certifier-operator/lib/charms/magma_orc8r_certifier/v0/cert_fluentd.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Cert Fluentd Library.
+
+This library offers ways of providing and consuming a fluentd certificate.
+
+To get started using the library, you just need to fetch the library using `charmcraft`.
```

Comment:
```suggestion
To get started using the library, you need to fetch the library using `charmcraft`.
```
Using words like `just` and `simply` is usually superfluous. This blog post is great on the subject: https://justsimply.dev/

Reply:
Removed.
Also update rootca lib.